src/sentry/api/endpoints/group_notes.py

```diff
@@ -57,14 +61,39 @@ def post(self, request, group):
             reason=GroupSubscriptionReason.comment,
         )
 
-        if mentions:
-            users = User.objects.filter(id__in=mentions)
-            for user in users:
-                GroupSubscription.objects.subscribe(
-                    group=group,
-                    user=user,
-                    reason=GroupSubscriptionReason.mentioned,
-                )
+        actors = Actor.resolve_many(mentions)
+        actor_mentions = seperate_resolved_actors(actors)
+
+        subscribed_user_ids = set()
+
+        for user in actor_mentions.get('users'):
+            GroupSubscription.objects.subscribe(
+                group=group,
+                user=user,
+                reason=GroupSubscriptionReason.mentioned,
+            )
+            subscribed_user_ids.add(user.id)
+
+        mentioned_teams = actor_mentions.get('teams')
+
+        mentioned_team_users = User.objects.filter(
+            sentry_orgmember_set__organization_id=group.project.organization_id,
+            sentry_orgmember_set__organizationmemberteam__team__in=mentioned_teams,
+            sentry_orgmember_set__organizationmemberteam__is_active=True,
+            is_active=True,
+        )
+
+        for user in mentioned_team_users:
+
+            if user.id in subscribed_user_ids:
```

Comment:
Do you need to do this anymore? Couldn't we just `exclude` them from the queryset above?

```python
mentioned_team_users = User.object.filter(...).exclude(id__in=subscribed_user_ids)
```

This whole thing feels a bit clunky imo. Since `subscribed_user_ids` itself is just derived from `actor_mentions` It seems like we could drop the variable entirely and just use:

```python
.exclude(id__in={u.id for u in actor_mentions['users']})
```

And avoid the extra bookkeeping.